packages/obonode/obojobo-chunks-heading/editor-registration.js

```diff
@@ -18,6 +23,21 @@ const Heading = {
 		emptyNode
 	},
 	plugins: {
+		onPaste(event, editor, next) {
+			const isHeading = isType(editor)
+			const transfer = getEventTransfer(event)
+
+			// If we are pasting standard text (not-html) then we simply
+			// put the text inside the existing heading node. If we don't
+			// do this the page will crash when pasting text into a blank
+			// heading node.
+			if (isHeading && transfer.type === 'text') {
```

Comment:
```suggestion
			if (isHeading && transfer.type !== 'fragment') {
```
fixes typo in @qwertynerd97 's suggestion above 'fragmentt'